.github/workflows/build-kube-fledged.yml

```diff
@@ -4,168 +4,176 @@ on:
-        description: 'kubefledged-cri-client dockerfile path'
+        description: "kubefledged-cri-client dockerfile path"
         required: true
         type: string
       dockerfile_operator:
-        description: 'kubefledged-operator dockerfile path'
+        description: "kubefledged-operator dockerfile path"
         required: true
-        type: string                
+        type: string
       dockerfile_webhook_server:
-        description: 'kubefledged-webhook-server dockerfile path'
+        description: "kubefledged-webhook-server dockerfile path"
         required: true
-        type: string                                
+        type: string
       golang_version:
-        description: 'Go version used during the build'
+        description: "Go version used during the build"
         required: true
         type: string
       operator_image_repo:
-        description: 'kubefledged-operator repository'
+        description: "kubefledged-operator repository"
         required: true
         type: string
       operator_target_platforms:
-        description: 'target platforms for operator multi-arch image'
+        description: "target platforms for operator multi-arch image"
         required: true
-        type: string        
+        type: string
       operatorsdk_version:
-        description: 'operator sdk release version'
+        description: "operator sdk release version"
         required: true
         type: string
       push_image:
-        description: 'whether or not to push built images to dockerhub'
+        description: "whether or not to push built images to dockerhub"
         required: true
-        type: boolean                
+        type: boolean
       release_version:
-        description: 'kube-fledged release version'
+        description: "kube-fledged release version"
         required: true
         type: string
       target_platforms:
-        description: 'target platforms for multi-arch images'
+        description: "target platforms for multi-arch images"
         required: true
-        type: string        
+        type: string
       webhook_server_image_repo:
-        description: 'kubefledged-webhook-server repository'
+        description: "kubefledged-webhook-server repository"
         required: true
         type: string
     secrets:
       dockerhub_user:
-        description: 'Dockerhub username'
+        description: "Dockerhub username"
         required: false
       dockerhub_pswd:
-        description: 'Dockerhub password'
-        required: false        
+        description: "Dockerhub password"
+        required: false
 
 jobs:
   build_kube_fledged:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ inputs.golang_version }}
+
+      - name: Run GoReleaser
```

Comment:
GoReleaser is to be used when releasing a particular version of kube-fledged. Following actions have to be performed when releasing:-
- Build the release Helm chart
- Publish the Helm chart in github.com/senthilrch/kubefledged-charts
- Add the Helm chart in senthilrch.github.io/kubefledged-charts
- Create a new release in github.com/senthilrch/kubefledged and update it with issues closed for the release
- Push the release tag to github.com/senthilrch/kubefledged

The GoReleaser action should be used in a separate workflow file named "release-kube-fledged.yml". The workflow in this file should automate the above steps and use the GoReleaser action to publish the release.